glmnet/linear.py

```diff
@@ -281,7 +288,7 @@ def _fit(self, X, y, sample_weight, relative_penalties):
                              relative_penalties,
                              coef_bounds,
                              max_features,
-                             max_features - 1,
+                             X.shape[1] + 1,
```

Comment:
This might be a dumb question, but previously this was equal to `max_features - 1`, which is the same as `X.shape[1]`. Is there a reason it's being incremented here?

Reply:
Hi, thanks for asking! Please see my longer comment above. In short, I think `ne` and `nx` were swapped before, and I'm superstitiously following the Fortran code's comment that `nx > ne`.  

If you'd prefer, I can set `nx = X.shape[1]`, since it seems like `nx` doesn't actually need to be strictly greater.

Reply:
Ah, that's great; I just should have read your initial message more closely. Thanks!